onnx/defs/gen_doc.py

```diff
@@ -32,126 +38,229 @@ def display_attr_type(v):
     return s
 
 
+def display_domain(domain):
+    if domain:
+        return "operator set '{}'".format(domain)
+    else:
+        return "default operator set"
```

Comment:
operator set 'ONNX may be better?

Reply:
How about "the default ONNX operator set"? You don't want to say operator set 'onnx' because that's *not* what you're supposed to put in the opset_version.